Sample-test/sample.js

```diff
@@ -1,8 +1,10 @@
-const addTodo = () => {
-  const newTodoInput = document.getElementById('newTodoInput');
-  let currentTodoList = document.getElementById('todoList').innerHTML;
-  currentTodoList += `<li>${newTodoInput.value}</li>`;
-  document.getElementById('todoList').innerHTML = currentTodoList;
-  newTodoInput.value = '';
-};
-document.getElementById('addTodoBtn').addEventListener('click', addTodo);
+// const addTodo = () => {
+//   const newTodoInput = document.getElementById('newTodoInput');
+//   let currentTodoList = document.getElementById('todoList').innerHTML;
+//   currentTodoList += `<li>${newTodoInput.value}</li>`;
+//   document.getElementById('todoList').innerHTML = currentTodoList;
+//   newTodoInput.value = '';
+// };
+// document.getElementById('addTodoBtn').addEventListener('click', addTodo);
+
+// export default addTodo;
```

Comment:
If these are not needed then we can remove this file

Reply:
It basically for pr of test setup to show how we suppose to write a test